package.json

```diff
@@ -1,18 +1,45 @@
 {
-  "name": "tao-wallet",
-  "version": "2.0.1",
-  "description": "",
-  "main": "index.js",
-  "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1"
-  },
-  "keywords": [],
-  "author": "",
-  "license": "ISC",
-  "dependencies": {
-    "@ln-markets/api": "^1.7.0",
-    "axios": "^0.27.2",
```

Comment:
I removed axios but forgot to set the node version in the engines field. I can do that tmrw